src/java/htsjdk/samtools/util/SamLocusIterator.java

```diff
@@ -75,12 +77,15 @@ public RecordAndOffset(final SAMRecord record, final int offset) {
 
     /**
      * The unit of iteration.  Holds information about the locus (the SAMSequenceRecord and 1-based position
-     * on the reference), plus List of ReadAndOffset objects, one for each read that overlaps the locus
+     * on the reference), plus List of ReadAndOffset objects, one for each read that overlaps the locus;
+     * two more List of ReadAndOffset objects includes reads that overlaps the locus with insertions/deletions
```

Comment:
this doesn't read quite right to me.  How about:
two more List_s_ of ReadAndOffset objects include reads that overlap the locus with insertions and deletions respectively.